openid-federation-1_0.xml

```diff
@@ -6733,42 +6736,72 @@ HTTP/1.1 302 Found
 	  as the <spanx style="verb">crit</spanx> value.
 	  Use of this claim is OPTIONAL.
 	</t>
+	<t>
+	  This claim is used in this specification in <xref target="entity-statement"/>
```

Comment:
claim-> Claim

Reply:
We're usually using "claim" in running text.